### PR TITLE
Remove safelinks from replies

### PR DESCRIPTION
**Pull request by jan-kiszka**
Rebased #4, superseding it. I specifically had to massage change_replies.js.

@nwhitehorn

### CODE REVIEW

**Comment by cphyc** [comment]
Does it work for in-text links as well?

**Comment by jan-kiszka** [comment]
It demangles the email I provided in https://github.com/cphyc/thunderbird_remove_safelinks/issues/6, e.g.

**Comment by nwhitehorn** [comment]
Works fine for me.

File: src/background.js [hunk 1]
```diff
@@ -2,12 +2,17 @@ let myPromises = [
...
         ],
     }),
     messenger.messageDisplayScripts.register({
         js: [
             { file: "/change_message.js" }
         ],
+    }),
+    messenger.composeScripts.register({
+        js: [
+            { file: "/decoders.js" },
```
Comment (nwhitehorn) [inline src/background.js:14]: You also need change_message.js here to adjust links in HTML messages.
```suggestion
            { file: "/decoders.js" },
            { file: "/change_message.js" },
```

**Comment by jan-kiszka** [comment]
Seems like https://github.com/cphyc/thunderbird_remove_safelinks/commits/change-in-replies is obsoleting this. Close it, @cphyc, @nwhitehorn?

**Comment by nwhitehorn** [comment]
This one works by itself for me, and the other one needs patches. Beyond that, I don't care.

**Comment by nwhitehorn** [comment]
Just wanted to ping about this -- I've been using it for a month now without issue.

**Pull request merged**